### PR TITLE
fix image aspect ration in article detail

### PR DESCRIPTION
**Pull request by miltontony**

### CODE REVIEW

**Comment by miltontony** [comment]
@Mitso @saeedmb8 review please

**Comment by smarzban** [comment]
👍

File: babycenter/templates/core/article_page.html [hunk 1]
```diff
@@ -22,7 +22,8 @@ <h2 class="l-indent heading heading--back heading{{section.get_effective_extra_s
   <h1 class="article__title">{{self.title}}</h1>
 
   {% if self.image %}
-  {% image self.image width-288 class="article__image" %}
+    {% image self.image width-288 as article_image %}
+    <img src="{{ article_image.url }}" class="article__image" />
```
Comment (peterp) [inline babycenter/templates/core/article_page.html:26]: May want to reproduce for the article itself?

Comment (miltontony) [inline babycenter/templates/core/article_page.html:26]: @peterp not sure I follow what you're suggesting?

Comment (peterp) [inline babycenter/templates/core/article_page.html:26]: @miltontony Sorry, check line 34, we also use images later on in the file.

Comment (miltontony) [inline babycenter/templates/core/article_page.html:26]: ahh...yes, those. Those are images within the body of the article. We'll need to test these out and see if they also suffer from this. I suspect they do.

**Pull request merged**